editor/src/clj/editor/collision_object.clj

```diff
@@ -34,14 +34,18 @@
             [editor.types :as types]
             [editor.validation :as validation]
             [editor.workspace :as workspace]
-            [schema.core :as s])
+            [schema.core :as s]
+            [util.murmur :as murmur])
   (:import [com.dynamo.gamesys.proto Physics$CollisionObjectDesc Physics$CollisionObjectType Physics$CollisionShape$Shape]
            [javax.vecmath Matrix4d Quat4d Vector3d]))
 
 (set! *warn-on-reflection* true)
 
 (def collision-object-icon "icons/32/Icons_49-Collision-object.png")
 
+; Shared with editor.atlas - move to graph?
+(g/deftype ^:private NameCounts {s/Str s/Int})
+
```

Comment:
It's fine here, but if we should move it, I think `editor.types` would be the place to put it.